neovim/api/buffer.py

```diff
@@ -26,6 +26,13 @@ class Buffer(Remote):
 
     _api_prefix = "nvim_buf_"
 
+    def __repr__(self):
+        """Get text representation of the buffer."""
+        return 'Buffer(number=%r, handle=%r)' % (
+            self.number,
```

Comment:
`number` and `handle` are synonymous. After this, we can just as well have a shared implementation in `Remote`.